src/FileBackedTaskManager.java

```diff
@@ -8,20 +8,22 @@
 import java.nio.file.Files;
 import java.io.FileReader;
 import java.io.Reader;
+import java.io.BufferedReader;
 import java.io.File;
+import java.util.List;
 
 public class FileBackedTaskManager extends InMemoryTaskManager  {
 
     private static final String HOME = System.getProperty("user.home");
-    File dir = new File("C:\\Users\\Igor\\java-kanban\\dbTaskManager");
 
-    File dbTaskManager = new File(dir, "dbTaskManager.csv");
+    File dir = new File("C:\\Users\\Igor\\java-kanban\\dbTaskManager");
+    File dbTaskManager = new File("dbTaskManager.csv");
     private int id = 0;
     private int lastEpicId = 0;
-    private HashMap<Integer, Task> singleTaskDesc = new HashMap<>();
-    private HashMap<Integer, Epic> epicTaskDesc = new HashMap<>();
-    private HashMap<Integer, SubTask> subTaskDesc = new HashMap<>();
-    InMemoryHistoryManager inMemoryHistoryManager = new InMemoryHistoryManager();
+    private HashMap<Integer, Task> singleTaskDesc;
+    private HashMap<Integer, Epic> epicTaskDesc;
+    private HashMap<Integer, SubTask> subTaskDesc;
+    InMemoryHistoryManager inMemoryHistoryManager;
 
     public FileBackedTaskManager(File dbTaskManager, int id, int lastEpicId, HashMap<Integer, Task> singleTaskDesc, HashMap<Integer, Epic> epicTaskDesc, HashMap<Integer, SubTask> subTaskDesc, InMemoryHistoryManager inMemoryHistoryManager) {
         this.dbTaskManager = dbTaskManager;
```

Comment:
```suggestion
    public FileBackedTaskManager(File dbTaskManager) {
        this. dbTaskManager = dbTaskManager;
        }
```

Конструктор по требованиям должен получиться примерно таким, все остальное: id, списки эпиков и остальных задач - инициализируется внутри менеджера. не совсем понятно зачем это     "File dir = new File("C:\\Users\\Igor\\java-kanban\\dbTaskManager");
    File dbTaskManager = new File("dbTaskManager.csv");"